api/models.py

```diff
@@ -395,9 +395,9 @@ def validate_params(cls, params: dict):
     @classmethod
     def translate_fields(cls, params: dict):
         """Translate regression parameters"""
-        translated = cls.translate_fields(params)
-        translated.update(cls._translate_object_ids(translated))
-        translated.update(cls._translate_timestamps(translated))
+        translated = Node.translate_fields(params)
+        translated.update(Node._translate_object_ids(translated))
+        translated.update(Node._translate_timestamps(translated))
```

Comment:
Actually maybe this `Regression.translate_fields()` method can just be dropped as it's not doing anything else than what the base `Node.translate_fields()` one is doing?

Reply:
Also, if you really wanted to call the base class this would be better:
```python
translated = super().translate_fields(params)
translated.update(cls._translate_object_ids(translated))
# etc.
```

Reply:
> Actually maybe this `Regression.translate_fields()` method can just be dropped as it's not doing anything else than what the base `Node.translate_fields()` one is doing?

By mistake, I changed the code for `_translate_object_ids` and `_translate_timestamps` methods.
It should be as-is to use `_OBJECT_ID_FIELDS` and `_TIMESTAMP_FIELDS` from `Regression` model rather than from `Node`.

Reply:
> Also, if you really wanted to call the base class this would be better:
> 
> ```python
> translated = super().translate_fields(params)
> translated.update(cls._translate_object_ids(translated))
> # etc.
> ```

It won't work as we need `<class 'api.models.Node'>` to be the class to instantiate the method and not `<class 'api.models.Regression'>`.